### ISSUE TITLE
License text still contains Microsoft as Licenser

### ISSUE THREAD

**Issue by matthias-mucha-at-sick** [eclipse-threadx/netxduo#287]
**Describe the bug**
The license.txt still contains Microsoft as Licenser. Shouldn't that be something related to the eclipse foundation?

**To Reproduce**
Take a look into the third line of license.txt

**Expected behavior**
Does not contain references to Microsoft

**Impact**
Internal release process is more difficult

**Comment by fdesbiens** [comment]
Hi @matthias-mucha-at-sick.

Thank you for raising this issue. 

The Eclipse Foundation owns ThreadX's intellectual property on behalf of the developer community. However, the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/eca/) does not include a license or assignment to the Eclipse Foundation. Unlike many other open source communities or projects, Eclipse simply wants you to license your contributions under the open source license(s) the project uses. So you (or your employer) will continue to own your code.

In other words, Microsoft still rightfully owns the copyrights on ThreadX. However, they cannot revert the open sourcing of ThreadX since we have a written record that they have agreed to provide their contributions of code and documentation under the MIT license.

I will close this issue for now. Feel free to continue the conversation as needed.
